framework_lib/src/touchpad.rs

```diff
@@ -28,6 +37,7 @@ pub fn print_touchpad_fw_ver() -> Result<(), HidError> {
                 let vid = dev_info.vendor_id();
                 let pid = dev_info.product_id();
                 let usage_page = dev_info.usage_page();
+                let hid_ver = dev_info.release_number();
```

Comment:
@aokblast could you please check if this value is populated with your changes? 
The kernel and hidapi on Linux don't expose that, except for USB. But windows does. 
I checked that the FreeBSD kernel does expose it, could you please make sure your hidapi changes do as well?
You can run with `-vvv` verbose logging and it'll be printed.

Reply:
For reference, we checked that it is populated on FreebBSD.